test/pages.js

```diff
@@ -2227,7 +2250,7 @@ describe('Pages', function() {
             patch._id
           );
         } catch (error) {
-          console.error(error);
+          assert(false);
```

Comment:
we could just remove the try/catch so that error would be thrown and the tests would fail